apps/src/templates/SchoolDataInputs.jsx

```diff
@@ -33,7 +33,7 @@ export default function SchoolDataInputs({
     detectedCountry === US_COUNTRY_CODE
   );
   const [isOutsideUS, setIsOutsideUS] = useState(
-    detectedCountry && detectedCountry !== US_COUNTRY_CODE
```

Comment:
While testing, I found a bug that if `usIp` was passed in as `false`, `isOutsideUS` would still return `false` because it would process `detectedCountry` as `''` (meaning the school name field wouldn't automatically show up). This handles the case now for when `usIp` is `false`, rather than treating `false` and `undefined` as the same for `usIp`.